src/style.css

```diff
@@ -1,3 +1,70 @@
+* {
+  margin: 0;
+
+  --bluecolor: #00acdc;
```

Comment:
Consider defining more CSS variables for other colors or frequently used values to enhance maintainability and consistency across the stylesheet.